test/mixer_test.py

```diff
@@ -1183,6 +1183,24 @@ def test_get_raw(self):
             with self.assertRaisesRegex(pygame.error, "mixer not initialized"):
                 snd.get_raw()
 
+    def test_copy(self):
+        """Test that the copy method would return correct values and etc"""
+        samples = b"abcdefgh"  # keep byte size a multiple of 4
+        a = mixer.Sound(buffer=samples)
+        a.set_volume(0.7)
+        b = a.copy()
+        self.assertEqual(a.get_raw(), b.get_raw())
+        self.assertEqual(a.get_volume(), b.get_volume())
+
+        class Amogus(mixer.Sound):
+            pass
+
+        c = Amogus(buffer=samples)
+        d = c.copy()
+        self.assertEqual(type(d), Amogus)
+        self.assertEqual(d.get_raw(), c.get_raw())
+        self.assertEqual(d.get_volume(), c.get_volume())
```

Comment:
I wonder if channels work properly?

Another test which plays a longer sound, does a copy, and then compares the number of channels would be good.

I'd expect the channels to not be copied? But I'm unsure... what do you think should happen to channels when the sound is copied?

Asserting the .get_num_channels() is zero in the other tests is probably a good idea.

Reply:
I think the channels should be left alone when a Sound is copied. I'm not sure if that's desirable though.